src/main/scala/gw/cda/api/outputwriter/JdbcOutputWriter.scala

```diff
@@ -326,10 +329,11 @@ private[outputwriter] class JdbcOutputWriter(override val outputPath: String, ov
       }
 
       // Build the sql Update statement to be used as a prepared statement for the Updates.
-      val colListForSetClause = latestChangeForEachID.columns.filter(_ != "id")
+      val colListForSetClause = latestChangeForEachID.columns
+        .filter(_ != "id")
+        .filter(_ != "hexseqvalue")
```

Comment:
```suggestion
        .filter(_ != "id" && _ != "hexseqval" )
```

Reply:
Won't resolve the second '!='. May be why I had them on separate lines.
![image](https://user-images.githubusercontent.com/64554153/121593167-3bae7880-ca01-11eb-8e31-caa77f46a30d.png)

leaving as is for now. will dig into it later.